tgui/packages/tgui/interfaces/Reflector.js

```diff
@@ -0,0 +1,201 @@
+import { useBackend } from '../backend';
+import { Box, Button, Flex, Stack, Icon, LabeledControls, Section, NumberInput, Table } from '../components';
+import { Window } from '../layouts';
+
+export const Reflector = (props, context) => {
+  const { act, data } = useBackend(context);
+  return (
+    <Window
+      title={data.reflector_name}
```

Comment:
prefer to deconstruct these before the return block so we're not using data.xxx,

`const { reflector_name, rotation_angle } = data;`